docs/integration.md

```diff
@@ -0,0 +1,248 @@
+
+## [Prerequisites](#prerequisites)
+
+This integration guide assumes you already have a web delivery that serves content & a mobile app is being setup for that  particular web delivery.
+
+## [Requirements](#requirements)
+
+#TODO - write requirements that the web delivery needs to satisfy in order for the module to work
+
+## [How to install](#how-to-install)
+> The module is designed to run in the browser. It can also be used in a server (node) environment via npm.
+
+### [Installation via script tag](#install-via-script-tag)
+```html
+<script src="https://unpkg.com/@forward-distribution/bridget">
+  // The module will bootstrap itself when imported, if it is in the correct context (mobile webview)
+</script>
+```
+Put the above script tags in between the <head> tags of your HTML page. It will attach a global `bridget` object. You can access it either via `bridget` or `window.bridget`.
+
+## [Alternative installation via NPM](#alternative-installation-via-npm)
+The module is available on NPM, if you’re using a front-end packager like Browserify or Webpack:
+
+Just run
+
+```
+npm install -g @forward-distribution/bridget
+```
+
+You can then `require` or `import` the lib like any other module.
+
+## [Internals](#internal)
+### Reacting to user actions
+Actions are something that a user performed on your website such as “*Clicked on a teaser*”, "*Navigated to a page/department*", or “*Attempted to share*”.
+
+In the context of your webpage, the browser itself or your custom JavaScript code handles these user events, and reacts to them accordingly. 
+
+In the mobile context, however, these actions need to be propagated and offloaded from the browser (webview) and sent onto the mobile app to be handled natively. 
+
+This is where you, as an integrator, will need to have bridget integrated, so these specific actions can  be handled (see [api reference](#api)).
+
+As an example, internally in the bridge, handling an `onclick` event originating from an `<a>` tag, would resemble something like:
+
+```js
+	onClickListener = e => {
+		e.preventDefault()
+		e.stopPropagation()
+        // data can come from anywhere; page meta, element attribute, data attributes, linked data.
+        // this particular example uses data attriutes
+		const { title, url } = some.meta.data
+		bridge.navigateToDoc({ url, title })
+    }
+```
+`onClickListener` is just an implementation detail, nothing you as an integrator should worry about; it is something `bridget` binds to the webpage for all elements or specific elements; in this case, it's for all anchor tags.
+
+# [API](#api)
+
+In the API segment of the integration guide we will define all of the specific changes the delivery should support in order for it to be compatible with `bridget` and to function within the WebView context.
+
+## Hijacking `onclick` events
```

Comment:
Again, internals with too much techy lingo about the internals used.

Do I as an integrator need to know this?
Don't think so. I need to know that you are doing the linking for me, due to the native navigation and that it works on clicks. That's all.